src/game/gui/text_render.c

```diff
@@ -318,6 +321,29 @@ static void text_render_len(const text_settings *settings, text_mode mode, int x
             }
 
             render_char_shadow_surface(settings, sur, mx + start_x, my + start_y);
```

Comment:
Please update the comment on line 317 that reads: `// Render character`
as this loop does not render the character, just the shadows.